ui/public/metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+  "versions": {
+    "main": "https://nomad-storybook-and-ui.vercel.app/storybook"
+  }
```

Comment:
i need to look into this comment https://github.com/hashicorp/cloud-ui/pull/1457#discussion_r724258881 

i'm thinking it may not apply to nomad, cause the product is self-hosted, not cloud?

Reply:
Oh snap, it definitely still applies. Everything in `/public` gets shrink-wrapped into the nomad binary via [go-bindata-assetfs](https://github.com/elazarl/go-bindata-assetfs).